specs/WindowControlsOverlayConfiguration.md

```diff
@@ -0,0 +1,156 @@
+
+This API is designed to be used in addition with the other non-client region APIs 
+and features. These include `app-region: drag`, and `IsNonClientRegionSupportEnabled`.
+
+# Conceptual pages (How To)
+Here is a concept doc on the window controls overlay: https://wicg.github.io/window-controls-overlay/#concepts. 
+This was written for the PWA counterpart  for this feature. From the perspective of
+HTML & Javascript layers, everything there applies in Webview2 as well.
+
+# Examples
+
+## Win32 C++
+```cpp 
+AppWindow::AppWindow() {
+    m_mainWindow = CreateWindowExW(
+        WS_EX_CONTROLPARENT, 
+        GetWindowClass(),
+        L""
+        WS_POPUPWINDOW, 
+        0,0, 800, 800, 
+        nullptr, nullptr, 
+        g_hInstance, nullptr);
+}
+
+void  AppWindow::OnCreateWebview2ControllerCompleted(HRESULT hr, ICoreWebview2Controller* controller) 
+{
+
+    wil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(m_controller->get_CoreWebView2(&coreWebView2));
+
+    wil::com_ptr<ICoreWebView2> coreWebView2_28;
+    CHECK_FAILURE(coreWebView2->QueryInterface(&coreWebView2_28));
+
+    wil::com_ptr<ICoreWebView2WindowControlsOverlaySettings> windowControlsOverlaySettings;
+    CHECK_FAILURE(coreWebView2_28->get_WindowControlsOverlaySettings(&wco_config));
+
+    CHECK_FAILURE(wco_config->put_IsEnabled(true));
+    COREWEBVIEW2_COLOR color {1, 0, 0, 225};
+    CHECK_FAILURE(wco_config->put_TitleBarBackgroundColor(color));
+}
+```
+## .NET C#
+```c#
+// WebView2 control is defined in the xaml
+// <wv2:WebView2 x:Name="webView" Source="https://www.microsoft.com/"/>
+public MainWindow() 
+{
+    InitializeComponent();
+    m_AppWindow.TitleBar.ExtendsContentIntoTitleBar = true;
+
+    CoreWebView2WindowControlsOverlaySettings config = Webview2.CoreWebivew2.WindowControlsOverlaySettings;
+    config.IsEnabled = true; 
+    config.color = Color.FromARGB(0, 0, 255);
+}
+```
+
+# API Details
+## Win32 C++
+```cpp
+/// Controller API used to configure the window controls overlay.
+/// To provide your app users with the best experience, it is important to handle webview 
+/// initialization errors appropriately. Provide your users with a way to close the window
+/// or restart the App.
+[uuid(101e36ca-7f75-5105-b9be-fea2ba61a2fd), object, pointer_default(unique)]
+interface ICoreWebView2_28 : IUnknown {
+  /// Gets the `WindowControlsOverlaySettings` object.
+  [propget] HRESULT WindowControlsOverlaySettings([out, retval] ICoreWebView2WindowControlsOverlaySettings** value);
+}
+
+/// This is the ICoreWebView2WindowControlsOverlaySettings
+[uuid(c9f7378b-8dbb-5445-bacb-08a3fdf032f0), object, pointer_default(unique)]
+interface ICoreWebView2WindowControlsOverlaySettings : IUnknown {
+    /// Gets the `Height` property.
+  [propget] HRESULT Height([out, retval] UINT32* value);
+
+
+  /// The `Height` property in raw screen pixels, allows you to set the height of the overlay and
```

Comment:
@bradp0721 Does it make sense to use raw screen pixels here vs view pixels?

Also, we should ensure that different DPI awareness doesn't create bugs with real raw pixels vs virtual DPI scaled raw pixels.